wm_extra/views.py

```diff
@@ -419,9 +419,11 @@ def geoexplorer2worldmap(config, map_obj, layers=None):
             if layer_config['local'] == True:
                 # WM local layer to process
                 if 'styles' not in layer_config:
-                    layer = Layer.objects.get(typename=layer_config['name'])
```

Comment:
@capooti This was a provisional fix to display maps. We haven't finished layers_styles migration table. However, as you said, some layers don't have style, causing a **List index out of range** in django. I'll send another pull request uncommenting the lines